Vagrantfile

```diff
@@ -61,6 +61,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision "ansible" do |ansible|
       ansible.playbook = "#{dir}/provisioning/playbook.yml"
       ansible.sudo = true
+      ansible.extra_vars = {
```

Comment:
This configuration is not needed, as Ansible picks up any configuration from config.yml automatically.